patches/minecraft/net/minecraft/item/Item.java.patch

```diff
@@ -386,10 +386,9 @@
 +     * @return  A ModelBiped to render instead of the default
 +     */
 +    @SideOnly(Side.CLIENT)
-+    @Nullable
 +    public net.minecraft.client.model.ModelBiped getArmorModel(EntityLivingBase entityLiving, ItemStack itemStack, EntityEquipmentSlot armorSlot, net.minecraft.client.model.ModelBiped _default)
```

Comment:
Is the package `@ParametersAreNonnullByDefault`? If no then I suggest to add `@Nonnull` to the arguments also.